lib/AST/TypeRepr.cpp

```diff
@@ -218,6 +218,15 @@ void AttributedTypeRepr::printAttrs(ASTPrinter &Printer,
     Printer.printSimpleAttr("@autoclosure") << " ";
   if (hasAttr(TAK_escaping))
     Printer.printSimpleAttr("@escaping") << " ";
+
+  for (auto customAttr : Attrs.getCustomAttrs()) {
```

Comment:
I wasn't sure whether it might make sense to try to specifically find the global actor attribute here and print it individually, rather than all of the custom attributes all at once. In the future when there are multiple kinds of attributes we might want to print each possible custom attribute kind in a stable order.